torq.q

```diff
@@ -616,7 +617,7 @@ if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 		[{[a].lg.o[`init;"attemping to run initialisation: ",-3!a];
```

Comment:
I think instead of if-else this would be better as an exit early check if `.proc.initlist` is empty. Something like:
```q
if[0=count .proc.initlist;:.lg.o[`init;"no initialisation functions found"]];
{[a].lg.o....
```
Avoids the need for a code block. Thoughts @jonnypress and @jonathonmcmurray?

Reply:
I agree this is much neater. Only caveat is that the '`$` conditional with code blocks' pattern is used in a lot of places in TorQ, so this may be inconsistent, but I think refactoring this way is more readable & my opinion would be that takes precedence.